backend/test_firestore.go

```diff
@@ -70,15 +76,17 @@ func GetUser(w http.ResponseWriter, r *http.Request) {
 	user := &firebase.User{FirebaseID: vars["id"]}
 	err := user.Get()
 	if err != nil {
-		w.WriteHeader(http.StatusNotFound)
 		log.Warnf("Failed to get event : %v", err)
 		http.Error(w,"Could not find user with the given id",http.StatusNotFound)
 		return
 	}
 
 	w.WriteHeader(http.StatusOK)
 	// marshal the data struct to JSON to send as response
-	json.NewEncoder(w).Encode(user)
+	err = json.NewEncoder(w).Encode(user)
+	if err != nil {
+		log.Warnf("There was an error sending the response")
```

Comment:
we might benefit from having more detailed log messages here (e.g. "error sending resp: couldn't marshal struct to json")